python_modules/libraries/dagster-looker/dagster_looker/api/dagster_looker_api_translator.py

```diff
@@ -130,11 +137,13 @@ def get_explore_asset_spec(
             lookml_model_name = check.not_none(lookml_explore.model)
             lookml_explore_name = check.not_none(lookml_explore.explore)
 
-            return AssetSpec(key=f"{lookml_model_name}::{lookml_explore_name}")
+            return AssetSpec(key=_clean_asset_name(f"{lookml_model_name}::{lookml_explore_name}"))
```

Comment:
nit: we can explicitly state that we want an underscore between the names instead of `::`.
```suggestion
            return AssetSpec(key=_clean_asset_name(f"{lookml_model_name}_{lookml_explore_name}"))
```